WEB/FE/src/components/LogIn/LogInContainer.tsx

```diff
@@ -40,11 +43,17 @@ const LogInContainer = ({}: LogInContainerProps): JSX.Element => {
     setModalDisabled(true);
     executeRecaptcha('LogInWithOTP')
       .then((reCaptchaToken: string) => loginWithOTP({ authToken, totp: TOTP, reCaptchaToken }))
-      .then(() => history.replace('/'))
+      .then((data: Promise<any>) => successLoginHandler(data))
```

Comment:
promise가 resolve된 상태라 any가 더 적합한 것 같습니다
```suggestion
      .then((data: any) => successLoginHandler(data))
```